test/e2e/_helpers.js

```diff
@@ -119,12 +114,14 @@ export async function createPersonFixture(overrides = {}) {
  *
  * @returns {string} - location id
  */
-const getRandomLocation = (locationType = 'police') => {
-  return faker.random.arrayElement(
-    locations
-      .filter(loc => loc.location_type === locationType)
-      .map(({ id }) => id)
-  )
+const getRandomLocation = async (locationType = 'police') => {
```

Comment:
I think we should still drop the default. With this the `else` currently won't ever fire.

Reply:
I think I hallucinated removing the default